### PR TITLE
INTYGFV-15056: Implementation of endpoint and new resourcelink to cater for new modal showed in DOI.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

**Comment by ChrilleSigerhed** [comment]
WIP - synk with frontend in progress

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCandidateMessageForCertificateFacadeServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    public GetCandidateMessageForCertificateFacadeServiceImpl(GetCertificateFacadeService getCertificateFacadeService,
+        CandidateDataHelper candidateDataHelper) {
+        this.getCertificateFacadeService = getCertificateFacadeService;
+        this.candidateDataHelper = candidateDataHelper;
+    }
+
+    @Override
+    public GetCandidateMessageForCertificateDTO get(String certificateId) {
+        final var certificate = getCertificateFacadeService.getCertificate(certificateId, false);
```
Comment (ChrilleSigerhed) [inline web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCandidateMessageForCertificateFacadeServiceImpl.java:50]: should pdl be true?

Comment (mww-sopra-steria) [reply web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCandidateMessageForCertificateFacadeServiceImpl.java:50]: No, it should be false. The code only retrieves the certificate to find if there are candidate certificates related to it.

**Review by mww-sopra-steria — APPROVED** [review]
Well done!

File: web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCandidateMessageForCertificateFacadeServiceImpl.java (hunk 1 above)
Comment (mww-sopra-steria) [reply web/src/main/java/se/inera/intyg/webcert/web/service/facade/impl/GetCandidateMessageForCertificateFacadeServiceImpl.java:50]: No, it should be false. The code only retrieves the certificate to find if there are candidate certificates related to it.

**Pull request merged**